tests/integration/auth.spec.coffee

```diff
@@ -34,7 +34,7 @@ describe 'SDK authentication', ->
 				resin.auth.authenticate(credentials)
```

Comment:
Preexisting, but I don't think the name of this test makes much sense. Looks like it's testing that `loginWithToken` works with api keys? We should move that to the `loginWithToken` tests, and rename it.

Reply:
There was no section for `loginWithToken` at all!
Moved the two tests that were using session tokens & api keys to the new `loginWithToken` section.